homeassistant/components/google_assistant/trait.py

```diff
@@ -326,15 +326,17 @@ def sync_attributes(self):
             response["colorModel"] = "hsv"
 
         if features & light.SUPPORT_COLOR_TEMP:
+
+            def convert(mireds):
+                if mireds is None:
+                    return None
+                return color_util.color_temperature_mired_to_kelvin(mireds)
+
```

Comment:
Somewhat of a workaround. A better solution might be to let the color util return None when input is None.

Reply:
Nah, color util shouldn't catch `None`s, it should blow up.